GA release and other updates (#4939)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/office-365-network-mac-perf-insights.md

> There are six specific network insights that may be shown for each office location:

Reviewer: ```suggestion
There are eight specific network insights that may be shown for each office location:
```

Reviewer 2: I count 7 network insights in the published doc plus one was added so this should say eight and not six.

Use of a non-optimal SharePoint Online service front door could be caused by network backhaul before the corporate network egress in which case we recommend local and direct network egress. It could also be caused by use of a remote DNS Recursive Resolver server in which case we recommend aligning the DNS Recursive Resolver server with the network egress.

## Poor connectivity to Microsoft Teams

This insight will be displayed if the network insights service detects that some users are experiencing poor call quality in Microsoft Teams.

### What does this mean?

This tests for UDP connectivity to the Microsoft Teams service front door. If this is blocked then Microsoft Teams may still work using TCP, but audio and video will be impaired.

### What should I do?

> Unblock, improve, or upgrade your network perimeter and internet connectivity so that it meets the Teams UDP media requirements. See [Microsoft Teams Media connectivity](https://docs.microsoft.com/en-us/Microsoft-365/Enterprise/office-365-network-mac-perf-onboarding-tool?view=o365-worldwide#media-connectivity-audio-video-and-application-sharing).

Reviewer: ```suggestion
Unblock, improve, or upgrade your network perimeter and internet connectivity so that it meets the Teams UDP media requirements. See [Microsoft 365 network connectivity test tool](/Microsoft-365/Enterprise/office-365-network-mac-perf-onboarding-tool).
```

Reviewer 2: Good change. Correction for doc link only.

> 

Reviewer: ```suggestion
```
